spec/features/admin/invoices/show_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'As an admin,' do
+  describe "When I visit an admin invoice show page" do
+    it "Then I see information related to that invoice including:
+      - Invoice id
+      - Invoice status
+      - Invoice created_at date in the format 'Monday, July 18, 2019'
+      - Customer first and last name" do
+
+      visit admin_invoice_path(Invoice.first)
+
+      expect(page).to have_content("Invoice ID: 1")
```

Comment:
Is invoice ID suppose to be checked twice?

Reply:
agreed. Looks like lines 13  14 are duplicated.

Reply:
good catch, thank you. I think I meant to switch one of these strings to "Invoice Items"